tests/UI/data/faker/product.ts

```diff
@@ -135,6 +156,18 @@ export default class ProductData {
     /** @type {string} Reference of the product */
     this.reference = productToCreate.reference || faker.string.alphanumeric(7);
 
+    /** @type {string|null} MPN of the product */
+    this.MPN = productToCreate.MPN || null;
+
+    /** @type {string|null} UPC of the product */
+    this.UPC = productToCreate.UPC || null;
+
+    /** @type {string|null} EAN13 of the product */
+    this.EAN13 = productToCreate.EAN13 || null;
+
+    /** @type {string|null} ISBN of the product */
+    this.ISBN = productToCreate.ISBN || null;
+
```

Comment:
Even if there are abbreviation, it should not be uppercase as other vars are on camelCase

Reply:
Done